public/video-ui/src/services/pandaReqwest.js

```diff
@@ -10,7 +11,8 @@ export function pandaReqwest(reqwestBody) {
         })
         .fail(err => {
           if (err.status == 419) {
-            const reauthUrl = '/reauth';
+            const store = getStore();
+            var reauthUrl = store.getState().config.reauthUrl;
```

Comment:
In the app.js file, config is set to either the returned config or an empty object, so this would account for that.